documentation/src/main/asciidoc/migration/index.asciidoc

```diff
@@ -45,9 +45,12 @@ To address that, either:
 === Outbox polling system tables
 
 If you use the incubating link:{hibernateSearchDocUrl}#coordination-outbox-polling[`outbox-polling` coordination strategy],
-you will be impacted by changes to the entity that represents the outbox event,
+you will be impacted by changes to the entities that represents the outbox event and agent,
 requiring database schema changes.
-A new column -  `created::timestamp` is introduced, `id` column changes the type from integer to varchar.
+For outbox event table a new column -  `created::timestamp` is introduced, `id` column changes the type from integer to varchar,
+except MSSQL where the id type becomes binary.
+For agent table `id` column changes the type from integer to varchar,
+except MSSQL where the id type becomes binary.
```

Comment:
I think the phrasing below is more correct, or at least would feel more natural:

```suggestion
For the outbox event table a new column `created::timestamp` is introduced, the `id` column changes its type from integer to varchar,
except MSSQL where the `id` type becomes binary.
For the agent table `id` column changes its type from integer to varchar,
except MSSQL where the `id` type becomes binary.
```